package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-VT0ZI6kZRdTh8YyJw3SMbYm/u+NqfsAxEpWO0Pf9sq8/e94WxxOpPKx9FR1FlyCtOVDNOQ+8ntlqFxiRc+r5qA==",
       "dev": true,
       "requires": {
-        "color-convert": "1.9.3"
+        "color-convert": "^1.9.0"
```

Comment:
Nit: Is committing changes here intentional?

Reply:
Oh, `npm install` script modifies this lock file automatically without my participation. I don't know how to disable this "feature".